libs/community/langchain_community/vectorstores/milvus.py

```diff
@@ -140,6 +142,7 @@ def __init__(
         replica_number: int = 1,
         timeout: Optional[float] = None,
         num_shards: Optional[int] = None,
+        enable_dynamic_field: Optional[bool] = False,
```

Comment:
can we just make this bool instead of Optional[bool]

Reply:
Yes, But i guess it will be a breaking changes for existing users.

Reply:
Anyways community is deprecated, so user need to do change code to integrate to new langchain , so we can change .